net/portmapper/portmapper.go

```diff
@@ -413,6 +415,11 @@ func (c *Client) createOrGetMapping(ctx context.Context) (external netaddr.IPPor
 		return netaddr.IPPort{}, err
 	}
 
+	pcpReqMapping := buildPCPRequestMappingPacket(myIP, localPort, prevPort, pcpMapLifetimeSec)
```

Comment:
I don't think we should race a PMP & PCP mapping. That's going to be a little chaotic.

Only send one or the other, preferring PMP for now.